### PR TITLE
added text

### PR DESCRIPTION
**Pull request by kwzhang62**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -7,6 +7,6 @@
     <title>Document</title>
 </head>
 <body>
-    
+    <p>Lorem ipsum, dolor sit amet consectetur adipisicing elit. Unde cum consectetur voluptate quos accusamus doloribus eveniet, similique neque. Vero voluptate, obcaecati aspernatur repudiandae voluptates, reiciendis atque ipsam ipsa expedita eius modi velit maiores odit magnam omnis assumenda esse porro. Molestiae.</p>
```
Comment (Thistle-Strum) [inline index.html:11]: Wow!  Really helpful paragraph.  I will definitely get to work on that!  Maybe add a header and some more paragraphs.

**Review by Thistle-Strum — CHANGES_REQUESTED** [review]
Again, this page is really coming along.  I think it's going to be big!

File: index.html (hunk 1 above)
Comment (kwzhang62) [reply index.html:11]: Put in a header and some more paragraphs. The page is looking real nice now!

File: index.html [hunk 2]
```diff
@@ -7,6 +7,9 @@
     <title>Document</title>
 </head>
 <body>
+    <header>Lorem is real Latin!</header>
```
Comment (Thistle-Strum) [inline index.html:10]: This is exactly what I was thinking!  This is huge.

File: index.html [hunk 3]
```diff
@@ -7,6 +7,9 @@
...
     <p>Lorem ipsum, dolor sit amet consectetur adipisicing elit. Unde cum consectetur voluptate quos accusamus doloribus eveniet, similique neque. Vero voluptate, obcaecati aspernatur repudiandae voluptates, reiciendis atque ipsam ipsa expedita eius modi velit maiores odit magnam omnis assumenda esse porro. Molestiae.</p>
+    <P>Lorem ipsum dolor sit amet consectetur adipisicing elit. A nam soluta deleniti. Omnis porro voluptatem eaque quam ipsa facilis libero.</P>
+    <p>Lorem ipsum dolor, sit amet consectetur adipisicing elit. Ipsa quisquam corrupti, in hic quaerat iusto modi explicabo cum qui? Praesentium, dicta minus ut iure culpa veniam cum ratione dolores beatae mollitia nisi modi. Recusandae earum dolor commodi ut saepe optio nulla beatae consectetur? Totam laudantium, dignissimos dolorum perferendis aperiam magnam!</p>
```
Comment (Thistle-Strum) [inline index.html:13]: These paragraphs made me cry.  I almost couldn't finish reading them.

**Review by Thistle-Strum — APPROVED** [review]
It's overwhelming to think about the impact of this content when the page goes live.

**Pull request merged**